qiskit_ibm_transpiler/utils.py

```diff
@@ -62,7 +61,7 @@ def random_permutation(n_qubits):
 
 
 def create_random_linear_function(n_qubits: int, seed: int = 123) -> LinearFunction:
-    rand_lin = lambda seed: LinearFunction(
+    rand_lin = lambda seed: LinearFunction(  # noqa:E731
         random_invertible_binary_matrix(n_qubits, seed=seed)
     )
 
```

Comment:
Let's fix this function. Tome it looks like we can simply do:
```
def create_random_linear_function(n_qubits: int, seed: int = 123) -> LinearFunction:
    return LinearFunction(random_invertible_binary_matrix(n_qubits, seed=seed))
```

Reply:
Thank you!